packages/@angular/cli/tasks/extract-i18n.ts

```diff
@@ -34,27 +36,32 @@ export const Extracti18nTask = Task.extend({
 
     const webpackCompiler = webpack(config);
     webpackCompiler.outputFileSystem = new MemoryFS();
+    const statsConfig = getWebpackStatsConfig(runTaskOptions.verbose);
 
     return new Promise((resolve, reject) => {
       const callback: webpack.compiler.CompilerCallback = (err, stats) => {
         if (err) {
           return reject(err);
         }
 
+        const json = stats.toJson('verbose');
+        if (runTaskOptions.verbose) {
```

Comment:
I'd prefer we didn't output bundle stats in `xi18n` since those bundles aren't actually generated on disk and we didn't  outputt this information for `xi18n` before. Can you leave just the warning/error logic please?